Content.Server/Singularity/EntitySystems/SingularityAttractorSystem.cs

```diff
@@ -0,0 +1,106 @@
+using System.Numerics;
+using Content.Server.Physics.Components;
+using Robust.Shared.Map;
+using Robust.Shared.Map.Components;
+using Robust.Shared.Physics;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Systems;
+using Robust.Shared.Timing;
+
+using Content.Shared.Singularity.Components;
+using Content.Shared.Singularity.EntitySystems;
+
+using Content.Server.Singularity.Components;
+
+namespace Content.Server.Singularity.EntitySystems;
+
+/// <summary>
+/// Handles singularity attractors.
+/// </summary>
+public sealed class SingularityAttractorSystem : EntitySystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IViewVariablesManager _vvManager = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly SharedPhysicsSystem _physics = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
```

Comment:
Please check your IDE as some of these look unused and should be showing up as warnings.

Reply:
fixed